server/controllers/api.py

```diff
@@ -336,7 +318,11 @@ def __init__(self):
 
     def add_score(self, user):
         args = self.parse_args()
-        backup = models.Backup.query.get(decode_id(args['bid']))
+        try:
+            bid = decode_id(args['bid'])
+        except (ValueError, TypeError):
```

Comment:
There's probably a better way to handle this - especially since this route uses reqparse.